README.md

```
@@ -203,48 +257,43 @@ pip install -r requirements.txt
 ## Contributing
 
 Contributions are welcome! Please feel free to submit a Pull Request.
 
 ## Troubleshooting
 
-**Issue: "Tesseract not found"**
+**Issue: "Tesseract not found"** (Windows/Linux only — macOS uses Vision, not Tesseract)
 - Make sure Tesseract is installed and in your system PATH
-- macOS: `brew install tesseract`
 - Linux: `sudo apt-get install tesseract-ocr`
 - Windows: `winget install --id UB-Mannheim.TesseractOCR` or download from [here](https://github.com/UB-Mannheim/tesseract/wiki)
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)